Source/Widgets/InfoBox/InfoBox.js

```diff
@@ -155,6 +177,32 @@ click: function () { closeClicked.raiseEvent(this); }');
         },
 
         /**
+         * Gets or sets the security settings for iFrames generated by the Infobox.
+         * This will disable sandboxing for the iFrame, and allow HTML to be in the title.
+         * Only do this if you are in complete control of the data that will be loaded into the InfoBox,
+         * otherwise it is a security issue.
+         * @memberof InfoBox.prototype
+         *
+         * @type {Boolean}
+         */
+        disableSecurity : {
```

Comment:
We shouldn't need this property on `InfoBox`.  Instead we should be using the `securityDisabled` property from the `viewModel`.  Try doing something like this:

`InfoBox`: 
```
frame.setAttribute('data-bind', 'style : { maxHeight : maxHeightOffset(40) , attr: {'sandbox': sandboxOptions}');
```

`InfoBoxViewModel`:
```
this.sandboxOptions= undefined;
knockout.defineProperty(this, 'sandboxOptions', function() {
    var disabled = this.securityDisabled ;
    return disabled ? false :  'allow-same-origin allow-popups allow-forms';
});
```